enhancements/kube-apiserver/audit-policy.md

```diff
@@ -317,6 +321,8 @@ With secure OAuth storage in-place (see "Logging of Token with Secure OAuth Stor
 therefore the `oauth-apiserver.openshift.io/secure-token-storage: true` annotation on the `apiservers.config.openshift.io/v1` resource,
 these policies are extended by allowing logging of `oauthaccesstokens` and `oauthauthorizetokens`.
 
+The user will enter in the name of the group and the Audit Policy Desired: `Default`, `WriteRequestBodies`, or `AllRequestBodies`. The group can be `all` or the group name the admin specifies.
```

Comment:
Would separate this into its own subsection of proposal. Alternative: write another enhancement just for the new change. But maybe we can get away by putting it here.

Reply:
Also this needs much more details about the API in the apiserver config resource.